.travis.yml

```diff
@@ -1,6 +1,5 @@
 ---
 language: go
 go:
-  - 1.3
-  - 1.4
+  - 1.7
```

Comment:
Why not latest?

Reply:
`tip` is there too:
```
go:
  - 1.7
  - tip
```

In my understanding it lets us say "we support everything 1.7+"

Reply:
Sounds good, thanks for the clarification.